itest/rust/src/object_tests/base_init_test.rs

```diff
@@ -197,3 +197,78 @@ fn base_postinit_refcounted() -> TaskHandle {
     assert_eq!(obj.get_reference_count(), 2);
     next_frame(move || assert_eq!(obj.get_reference_count(), 1, "eventual dec-ref happens"))
 }
+
+fn make_mesh_arrays() -> Array<Variant> {
+    let mut arrays = Array::new();
+    arrays.resize(
+        mesh::ArrayType::MAX.ord().try_into().unwrap(),
+        &Variant::nil(),
+    );
```

Comment:
Also here, [`ENUMERATOR_COUNT`](https://godot-rust.github.io/docs/gdext/master/godot/obj/trait.IndexEnum.html#associatedconstant.ENUMERATOR_COUNT):
```suggestion
    arrays.resize(
        mesh::ArrayType::ENUMERATOR_COUNT,
        &Variant::nil(),
    );
```